### PR TITLE
Added abramov module on my branch

### PR DESCRIPTION
**Pull request by Retttro**

### CODE REVIEW

**Review by shevchukm — COMMENTED** [review]
:+1:

**Comment by VitaliiChverenko** [comment]
+++

File: abramov_test.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require 'minitest/autorun'
+require './abramov.rb'
+
+class TestAbramovTasks < Minitest::Unit::TestCase
+include Abramov
```
Comment (okohuch) [inline abramov_test.rb]: This file is not readable and not clear that class contains the code. Indent the code inside the class!

File: abramov_test.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+def test_task_88v
+  assert_equal(28, task_88v(82))
+end
+def test_task_329
```
Comment (okohuch) [inline abramov_test.rb]: From https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md:

Use empty lines between defs and to break up a method into logical paragraphs.

**Comment by viktoriia-bilous** [comment]
1. Take a look at [this](http://ruby-doc.org/core-2.4.0/Integer.html#method-i-digits) method, could be useful
2. Please add screenshots of rubocop and test run result

**Comment by Retttro** [comment]
image
Tests passing image

File: abramov_test.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+require 'minitest/autorun'
```
Comment (okohuch) [inline abramov_test.rb:1]: I see you did not update it. The file should look like:

```ruby
require 'minitest/autorun'
require './abramov.rb'

class TestAbramovTasks < Minitest::Unit::TestCase
  include Abramov
    
  def test_task_88v
    assert_equal(28, task_88v(82))
  end
    
  def test_task_329
    assert_equal([6, 15, 24, 33, 42, 51, 60, 105, 114], task_329(123, 36))
  end
    
  def test_task_562
    assert_equal(false, task_562(555))
  end
end

```

Comment (okohuch) [reply abramov_test.rb:1]: https://github.com/bbatsov/ruby-style-guide

Comment (Retttro) [reply abramov_test.rb:1]: Oh, I mistakenly posted screen of another test file,original file that on git I have updated

**Pull request closed without merging**